doc/source/index.rst

```diff
@@ -40,17 +40,202 @@ An example of such a callable class used in trollflow2 is the
 `class:FilePublisher`.
 
 
-Available plugins
-+++++++++++++++++
+Main plugins
+++++++++++++
 
 .. currentmodule:: trollflow2.plugins
```

Comment:
My hope was that this would include the docstrings of the different plugins here. Obviously this isn't working according to the latest version of the docs published on rtd.

So either we make this work or we remove it and use your text below. Or we use what you wrote as docstrings :)